docs/manage/configuration.md

```diff
@@ -294,6 +294,19 @@ In the `Raw JSON` configuration, you will see the fragment ID in the `fragments`
 
 For an example of adding a fragment to a robot, see the [Viam Rover fragment tutorial](/try-viam/rover-resources/rover-tutorial-fragments/).
 
+### Modify the config of a robot that uses a fragment
+
+When you modify a fragment, those changes are pushed to all robots that use that fragment.
+If you need to modify the config of just one robot that uses a fragment you can do the following:
+
+1. Go to the **Fragments** subtab of the **Config** tab.
+2. Click **Remove** next to the fragment.
+3. Select and copy the contents of the fragment in the box on the right side of the **Fragments** subtab.
+5. Toggle to **Raw JSON** mode.

```

Comment:
I think a link here on raw json would also be good

Reply:
I could link to the "The Config tab" section but it's not very raw JSON-specific so I think it would be more distracting than helpful